package.json

```
@@ -51,9 +51,12 @@
     "start:graph-worker:dev": "dotenvx run -f .env.graph -- nest start graph-worker --watch",
     "start:graph-worker:debug": "dotenvx run -f .env.graph -- nest start graph-worker --debug --watch",
     "generate:openapi:account": "dotenvx run -f env-files/account.template.env -- nest start account-api -- --writeOpenApi",
+    "generate:openapi:account-worker": "dotenvx run -f env-files/account.template.env -- nest start account-worker -- --writeOpenApi",
     "generate:openapi:content-publishing": "dotenvx run -f env-files/content-publishing.template.env -e ENVIRONMENT=dev -- nest start content-publishing-api -- --writeOpenApi",
+    "generate:openapi:content-publishing-worker": "dotenvx run -f env-files/content-publishing.template.env -e ENVIRONMENT=dev -- nest start content-publishing-worker -- --writeOpenApi",
     "generate:openapi:content-watcher": "dotenvx run -f env-files/content-watcher.template.env -- nest start content-watcher -- --writeOpenApi",
     "generate:openapi:graph": "dotenvx run -f env-files/graph.template.env -- nest start graph-api -- --writeOpenApi",
+    "generate:openapi:graph-worker": "dotenvx run -f env-files/graph.template.env -- nest start graph-worker -- --writeOpenApi",
     "generate:openapi": "npm run generate:openapi:account ; npm run generate:openapi:content-publishing ; npm run generate:openapi:content-watcher ; npm run generate:openapi:graph",
```

need to add workers to this
```suggestion
    "generate:openapi": "npm run generate:openapi:account ; npm run generate:openapi:account-worker ; npm run generate:openapi:content-publishing ; npm run generate:openapi:content-publishing-worker ; npm run generate:openapi:content-watcher ; npm run generate:openapi:graph ; npm run generate:openapi:graph-worker",
```